2024-08-13 19:26:46,055 DEBUG    root            Loaded Command Group: ['gcloud', 'compute']
2024-08-13 19:26:46,058 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'firewall_rules']
2024-08-13 19:26:46,068 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'firewall-rules', 'create']
2024-08-13 19:26:46,183 DEBUG    root            Running [gcloud.compute.firewall-rules.create] with arguments: [--allow: "['tcp:8080']", --description: "Allow port 8080 traffic", --direction: "INGRESS", --priority: "1000", --project: "still-totality-432312-r8", --target-tags: "['http-server']", NAME: "allow-http-8080"]
2024-08-13 19:26:46,243 INFO     ___FILE_ONLY___ Creating firewall...
2024-08-13 19:26:46,247 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-08-13 19:26:46,582 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/still-totality-432312-r8/global/firewalls?alt=json HTTP/1.1" 409 None
2024-08-13 19:26:46,587 DEBUG    root            Chosen display Format:list[title="Could not fetch resource:",always-display-title]
2024-08-13 19:26:46,587 DEBUG    root            (gcloud.compute.firewall-rules.create) Could not fetch resource:
 - The resource 'projects/still-totality-432312-r8/global/firewalls/allow-http-8080' already exists
Traceback (most recent call last):
  File "/snap/google-cloud-cli/260/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/260/lib/googlecloudsdk/calliope/backend.py", line 822, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/260/lib/surface/compute/firewall_rules/create.py", line 142, in Run
    return client.MakeRequests([(client.apitools_client.firewalls, 'Insert',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/260/lib/googlecloudsdk/api_lib/compute/client_adapter.py", line 145, in MakeRequests
    utils.RaiseToolException(
  File "/snap/google-cloud-cli/260/lib/googlecloudsdk/api_lib/compute/utils.py", line 139, in RaiseToolException
    RaiseException(problems, calliope_exceptions.ToolException, error_message)
  File "/snap/google-cloud-cli/260/lib/googlecloudsdk/api_lib/compute/utils.py", line 148, in RaiseException
    raise exception(
googlecloudsdk.calliope.exceptions.ToolException: Could not fetch resource:
 - The resource 'projects/still-totality-432312-r8/global/firewalls/allow-http-8080' already exists

2024-08-13 19:26:46,599 ERROR    root            (gcloud.compute.firewall-rules.create) Could not fetch resource:
 - The resource 'projects/still-totality-432312-r8/global/firewalls/allow-http-8080' already exists

